### DISCUSSION TITLE
Can you help me mine filecoin ?  How else can I download and continue?

### DISCUSSION THREAD

**Discussion by hellblazerr** [filecoin-project/community#682] [Q&A]
wget  https://snapshots.mainnet.filops.net/minimal/latest
--2023-12-09 01:27:35--  https://snapshots.mainnet.filops.net/minimal/latest
snapshots.mainnet.filops.net (snapshots.mainnet.filops.net) çözümleniyor... 3.85.178.213, 54.225.120.193, 52.72.160.163
snapshots.mainnet.filops.net (snapshots.mainnet.filops.net)|3.85.178.213|:443 bağlanılıyor... bağlantı kuruldu.
HTTP isteği gönderildi, cevap bekleniyor... 302 Found
Yer: https://pub-fd31751bcb69400eb39e694385c19457.r2.dev/minimal/3458400_2023_12_08T18_00_00Z.car.zst [izleyen]
--2023-12-09 01:27:37--  https://pub-fd31751bcb69400eb39e694385c19457.r2.dev/minimal/3458400_2023_12_08T18_00_00Z.car.zst
pub-fd31751bcb69400eb39e694385c19457.r2.dev (pub-fd31751bcb69400eb39e694385c19457.r2.dev) çözümleniyor... 2606:4700::6812:323, 2606:4700::6812:223, 104.18.3.35, ...
pub-fd31751bcb69400eb39e694385c19457.r2.dev (pub-fd31751bcb69400eb39e694385c19457.r2.dev)|2606:4700::6812:323|:443 bağlanılıyor... bağlantı kuruldu.
SSL connection cannot be.

**Comment by wolfduan** [comment]
github

**Reply by hellblazerr** [reply]
Thank you for the answer. can you give a link? Does it have anything to do with my region?
